collector/nginx_plus.go

```diff
@@ -582,6 +586,8 @@ func (c *NginxPlusCollector) Collect(ch chan<- prometheus.Metric) {
 				prometheus.GaugeValue, upstreamServerStates[peer.State], labelValues...)
 			ch <- prometheus.MustNewConstMetric(c.streamUpstreamServerMetrics["active"],
 				prometheus.GaugeValue, float64(peer.Active), labelValues...)
+				ch <- prometheus.MustNewConstMetric(c.streamUpstreamServerMetrics["limit"],
```

Comment:
@StevenHessing Can you address this listing issue? The change looks good otherwise